### PR TITLE
Added container operation weights and OneOf documentation

### PR DESCRIPTION
**Pull request by shijianjian**
#1159

### CODE REVIEW

**Review by edgarriba — COMMENTED** [review]
What are the other possibilities to show of random_apply? Could we add some "weights" to sample the distribution?

**Comment by shijianjian** [comment]
> What are the other possibilities to show of random_apply? Could we add some "weights" to sample the distribution?

It's a good one. I will add it.

File: kornia/augmentation/container/augment.py [hunk 1]
```diff
@@ -102,6 +101,33 @@ class AugmentationSequential(ImageSequential):
         >>> out = aug_list(input, input, bbox, points)
         >>> [o.shape for o in out]
         [torch.Size([1, 2, 3, 5, 6]), torch.Size([1, 2, 3, 5, 6]), torch.Size([1, 2, 4, 2]), torch.Size([1, 2, 1, 2])]
+
+    Perform `OneOf <https://albumentations.ai/docs/api_reference/core/composition/#albumentations.core.composition.OneOf>`__
```
Comment (edgarriba) [inline kornia/augmentation/container/augment.py:105]: I wouldn't refer to other libraries

**Review by edgarriba — APPROVED** [review]
LGTM

File: kornia/augmentation/container/augment.py [hunk 2]
```diff
@@ -102,6 +101,33 @@ class AugmentationSequential(ImageSequential):
...
+    transformation with ``random_apply=1`` and ``random_apply_weights`` in ``AugmentationSequential``.
```
Comment (shijianjian) [inline kornia/augmentation/container/augment.py:106]:
```suggestion
    Perform ``OneOf`` transformation with ``random_apply=1`` and ``random_apply_weights`` in ``AugmentationSequential``.
```

File: kornia/augmentation/container/image.py [hunk 3]
```diff
@@ -68,6 +70,23 @@ class ImageSequential(SequentialBase):
         >>> out2, lab2 = aug_list(input, label=label, params=aug_list._params)
         >>> torch.equal(out[0], out2[0]), torch.equal(out[1], out2[1]), torch.equal(lab[1], lab2[1])
         (True, True, True)
+
+    Perform `OneOf <https://albumentations.ai/docs/api_reference/core/composition/#albumentations.core.composition.OneOf>`__
+    transformation with ``random_apply=1`` and ``random_apply_weights`` in ``ImageSequential``.
```
Comment (shijianjian) [inline kornia/augmentation/container/image.py:75]:
```suggestion
    Perform ``OneOf`` transformation with ``random_apply=1`` and ``random_apply_weights`` in ``ImageSequential``.
```

File: kornia/augmentation/container/patch.py [hunk 4]
```diff
@@ -86,6 +86,28 @@ class PatchSequential(ImageSequential):
         >>> out1 = seq(input, params=seq._params)
         >>> torch.equal(out, out1)
         True
+
+    Perform `OneOf <https://albumentations.ai/docs/api_reference/core/composition/#albumentations.core.composition.OneOf>`__
+    transformation with ``random_apply=1`` and ``random_apply_weights`` in ``PatchSequential``.
```
Comment (shijianjian) [inline kornia/augmentation/container/patch.py:91]:
```suggestion
    Perform ``OneOf`` transformation with ``random_apply=1`` and ``random_apply_weights`` in ``PatchSequential``.
```

File: kornia/augmentation/container/video.py [hunk 5]
```diff
@@ -74,6 +74,24 @@ class VideoSequential(ImageSequential):
         >>> out2, lab2 = aug_list(input, label, params=aug_list._params)
         >>> torch.equal(output, out2)
         True
+
+    Perform `OneOf <https://albumentations.ai/docs/api_reference/core/composition/#albumentations.core.composition.OneOf>`__
+    transformation with ``random_apply=1`` and ``random_apply_weights`` in ``VideoSequential``.
```
Comment (shijianjian) [inline kornia/augmentation/container/video.py:79]:
```suggestion
    Perform ``OneOf`` transformation with ``random_apply=1`` and ``random_apply_weights`` in ``VideoSequential``.
```

**Review by edgarriba — APPROVED** [review]
LGTM

**Pull request merged**